.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+exclude = venv/, .ve/, output_dir
+max-line-length = 119
```

Comment:
In other repos we have a list of ignore. Don't you want to add them here too?
`ignore = E123,E128,E265,E501,W601,W293`